src/components/views/TableCellContextMenu.tsx

```diff
@@ -118,8 +148,16 @@ export const ContextMenu: React.FC<{
         <PlusIcon /> Insert {columns} to the right
       </ActionButton>
       <Separator />
+      <ActionButton
+        disabled={!headerDir}
+        onClick={() => headerDir && runCommand(addHeaderRow(headerDir))}
+      >
+        <PlusIcon /> Insert header row {headerDir || 'above'}
```

Comment:
Based on the above comment this then could be `Insert header row ${headerPosition}`

Reply:
updated